.github/workflows/reusable-tests.yaml

```diff
@@ -365,6 +365,49 @@ jobs:
       # - run: cd "$(mktemp -d)" && anchor init hello-anchor && cd hello-anchor && yarn link @coral-xyz/anchor && yarn && anchor test && yarn lint:fix
       # - uses: ./.github/actions/git-diff/
 
+  test-anchor-compatibility:
```

Comment:
It would be better to add this in the `test-programs` [section](https://github.com/coral-xyz/anchor/blob/ddcb3b826042891b6e2167e0d1559d8d053086c8/.github/workflows/reusable-tests.yaml#L368) as it's much shorter and requires less duplication for CI code.

Reply:
Done!